modules/EED_SquareOnsite.module.php

```diff
@@ -106,4 +106,21 @@ public static function cancelSquareOrder(EE_Transaction $transaction)
             $CancelOrder->cancel($orderId, $orderVersion);
         }
     }
+
+
+    /**
+     * Gets the country ISO by its name
+     *
+     * @param string $country_name
+     * @return string
+     * @throws EE_Error|ReflectionException
+     */
+    public static function getCountryIsoByName(string $country_name): string
+    {
+        $country_model = EEM_Country::instance();
+        $country = method_exists($country_model, 'get_country_by_name')
+            ? $country_model->get_country_by_name($country_name)
+            : $country_model->get_one([['CNT_name' => $country_name]]);
+        return $country instanceof EE_Country ? $country->get('CNT_ISO') : '';
```

Comment:
just an FYI that the `CNT_ISO` can also be retrieved by calling `ID()` since it is used as the primary key for that model

```suggestion
        return $country instanceof EE_Country ? $country->ID() : '';
```